src/main.rs

```diff
@@ -125,6 +125,7 @@ fn main() -> Result<()> {
             );
             println!("\n{:<20} {}", "Config path", config_path.as_str()?);
             println!("{:<20} {:?}", "Theme path", config_file.theme);
+            println!("{:<20} {}", "Temp path", std::env::temp_dir().display());
```

Comment:
I can move this to a separate review, this was an accidental commit. But just a heads up that the temp path on macos is not `/tmp` and its a weird one, and I'd imagine windows is probably similarly weird so I figured this would be a good addition to `DebugInfo`.